google/cloud/spanner/query_options.h

```diff
@@ -30,6 +30,9 @@ inline namespace SPANNER_CLIENT_NS {
  * These QueryOptions allow users to configure features about how their SQL
  * queries executes on the server.
  *
+ * Note: If you add an attribute here, remember to update the implementation
```

Comment:
Ugh... that will make it to the public documentation too... Can you add the comment to a `.cc` file or to the beginning of the `private:` section?  If not, can you add it after the Doxygen block as non-Doxygen comment?

Reply:
There is no `.cc` for this interface, so `private:` section it is.